utils/digitemp/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2009-2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+# Comment: This makefile is based on the oldpackage makefile by acinonyx / nbd
+# https://dev.openwrt.org/browser/packages/utils/digitemp/Makefile
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=digitemp
+PKG_VERSION:=3.7.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/cvbkf/digitemp.git
```

Comment:
This is not original upstream? Why?
Why not using release tarball, but the repo as source directly. Please mention this in your commit description.
In this case, also change the PKG_VERSION so that it reflects the usage of a specific commit - it surely differs from the version 3.7.1.

Reply:
Mmh. I actually forgot why I did that. Let me check if the upstream version works.

Reply:
@mhei yeah it worked. see my latest changeset.